.github/workflows/pull-request.yml

```
@@ -42,7 +44,16 @@ jobs:
 
       - name: Get changed TypeScript files
         id: changed-files
-        uses: tj-actions/changed-files@v45
+        run: |
+          ONLY_CHANGED=false
+          ALL_CHANGED_FILES=$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} | tr '\n' ' ')
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)